src/app/components/json-editor-page/json-editor-page.component.ts

```diff
@@ -23,7 +23,8 @@ export class JsonEditorPageComponent {
       productName: {
         description: 'Name of the product',
         type: 'string',
-        examples: ['Apples', 'Oranges']
+        examples: ['Apples', 'Oranges'],
+        isNotEmpty: true
```

Comment:
Hmm, then every property need to have this defined in the schema to show the validation error?

Reply:
Yes... for now.  Unless we decide that require is always `isNotEmpty`.